include/seqan3/alphabet/views/translate_join.hpp

```diff
@@ -0,0 +1,393 @@
+                      "std::ranges::random_access_range.");
+        static_assert(nucleotide_alphabet<std::ranges::range_reference_t<std::ranges::range_reference_t<urng_t>>>,
+                      "The range parameter to views::translate_join must be over a range over elements of "
+                      "seqan3::nucleotide_alphabet.");
+
+        return detail::view_translate_join{std::forward<urng_t>(urange), tf};
+    }
+
+    //!\brief This adaptor is usable without setting the frames parameter in which case the default is chosen.
+    template <std::ranges::range urng_t>
+    constexpr friend auto operator|(urng_t && urange, translate_join_fn const & me)
+    {
+        return me(std::forward<urng_t>(urange));
+    }
+};
+
+} // namespace seqan3::detail
+
+// ============================================================================
+//  translate (adaptor object)
+// ============================================================================
+
+namespace seqan3::views
+{
+
+/*!\name Alphabet related views
+ * \{
+ */
+
+/*!\brief A view that translates nucleotide into aminoacid alphabet with 1, 2, 3 or 6 frames. Input and output range are always two-dimensional.
+ * \tparam urng_t The type of the range being processed.
+ * \param[in] urange The range being processed. Needs to be a range of ranges (two-dimensional).
+ * \param[in] tf A value of seqan3::tanslation_frames that indicates the desired frames.
+ * \returns A range of ranges containing frames with aminoacid sequence. See below for the properties of the returned range.
+ * \ingroup views
+ *
+ * \details
+ *
+ * \header_file{seqan3/alphabet/views/translate_join.hpp}
+ *
+ * This view can be used to translate nucleotide sequences into aminoacid sequences (see translation_frames for possible combination of frames).
+ * This view only operates on two-dimensional input (range of ranges) and outputs a range of ranges no matter the number of input sequences or
+ * the number of translation frames given. Therefore, it has the same capabilities as the standard view_translate but concatenates the different
+ * frames of the different input sequences rather than having a separate range for each input sequence containing the translated frames. In the output,
+ * frames are ordered in a way, that all requested frames are listed per sequence directly after each other in the order of input sequences.
+ * improved and efficient downstream post-processing if needed. However, the index of a frame for a specific sequence needs to be calculated via
+ * modulo operations in this case. The i-th frame of the j-th sequence can be calculated by n = (i * s) + j, where s is the number of frames used
+ * for translation (index starting at zero).
+ *
+ * In short, this views behaves the same as:
+ *
+ * TODO: make this a snippet
+ * ```cpp
+ * std::vector<std::vector<dna5>> vec {...};
+ * auto v = vec | views::translate | views::join;
+ * ```
```

Comment:
Is the snippet in the later example not enough and this remains a note?
 -> \snippet test/snippet/alphabet/views/translate_join.cpp example